### ISSUE TITLE
CssSyntaxError for shadowOffset

### ISSUE THREAD

**Issue by cihati** [kristerkari/stylelint-react-native#17]
I get
```Missed semicolon   CssSyntaxError```
for stuff like
```shadowOffset: { width: 0, height: 1 };```

I believe, this is the correct way to use `shadowOffset`, with the comma (searching in Google/Github codes suggest the same). Any ideas?

The larger code (I use styled-components with React Native):

```
const SaveItemWrapper = styled(View)`
  justifyContent: flex-start;
  zIndex: 2;
  height: 89;
  position: absolute;
  bottom: 0;
  left:0;
  right:0;
  shadowOffset: { width: 0, height: 1 };
  shadowOpacity: 0.1;
  shadowRadius: 4;
  elevation: 4;
  paddingTop: 13;
`
```

**Comment by kristerkari** [comment]
Thanks @cihati!

camel case is not used in CSS, and it's not valid CSS. Could you try to use `box-shadow` instead?

**Comment by cihati** [comment]
thanks for the answer @kristerkari = ) `justifyContent`, `shadowRadius`, etc are also camel case but stylelint does not complain about them. Also, many people use `shadowOffset` for RN + Styled Components ( see https://github.com/search?l=JavaScript&q=%2B%22shadowOffset%22+%2B%22styled-components%22+%2B%22react-native%22&type=Code )

Some possibly relevant discussion: https://github.com/styled-components/styled-components/issues/709

**Comment by kristerkari** [comment]
I'm not sure how strict styled components is with it's CSS, but since it is regular CSS, you should use kebab case, and not camel case.

Camel case is used when writing the styles in Javascript objects.

**Comment by kristerkari** [comment]
so, to conclude, only enable the `css-property-no-unknown` if you use styled-components and remember to write the properties in kebab case and not camel case.

I added a unit test to verify that there is a warning for camel case:
a4de02680d9744be2d8d79396bc78a4da6e03d05

I will release a new version soon with a few small fixes.
